content/docs/1.1.0/monitoring/metrics.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Longhorn Metrics for Monitoring
+weight: 2
+---
+## Volume
+
+| Name | Description  | Example |
+|---|---|---|
+| longhorn_volume_usage_bytes | Used storage space in bytes for this volume | longhorn_volume_usage_bytes{node="worker-2",volume="testvol"} 1.1917312e+08 |
```

Comment:
We changed the metric name `longhorn_volume_usage_bytes` to `longhorn_volume_actual_size_bytes` to avoid the confusion between the size of the filesystem (usage_bytes) vs used space by Longhorn block device (actual size). The change was made at this PR https://github.com/longhorn/longhorn-manager/pull/689. Sheng suggested the change and I think it makes sense. How do you think?